### PR TITLE
Document Proxy-Wasm ABI v0.1.0.

### PR DESCRIPTION
**Pull request by PiotrSikora**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+- `proxy_get_header_map_value` now returns `NOT_FOUND` instead of `OK` with
+  an empty value for non-existing keys.
+
+- Replaced `proxy_continue_request` with `proxy_continue_stream(HttpRequest)`.
+
+- Replaced `proxy_continue_response` with `proxy_continue_stream(HttpResponse)`.
+
+- Replaced `proxy_get_configuration` with `proxy_get_buffer_bytes(VmConfiguration)`
+  and `proxy_get_buffer_bytes(PluginConfiguration)`.
```
Comment (mpwarres) [inline CHANGELOG.md:28]: nit: `VM_CONFIGURATION` and `PLUGIN_CONFIGURATION`

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+### Changed
+
+- `proxy_on_request_headers` and `proxy_on_response_headers` added `end_of_stream`
+  flag as the 3rd argument.
+
+- `proxy_get_header_map_value` now returns `NOT_FOUND` instead of `OK` with
+  an empty value for non-existing keys.
+
+- Replaced `proxy_continue_request` with `proxy_continue_stream(HttpRequest)`.
```
Comment (mpwarres) [inline CHANGELOG.md:23]: nit: `HTTP_REQUEST` and `HTTP_RESPONSE`

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## 0.2.0 - 2020-08-03
+
+### Added
```
Comment (mpwarres) [inline CHANGELOG.md:9]: Is it worth mentioning the addition of `proxy_stream_type_t`, or not really since that is just supplementary to added functions?

**Review by mpwarres — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (PiotrSikora) [reply CHANGELOG.md]: Good catch, thanks!

File: CHANGELOG.md (hunk 2 above)
Comment (PiotrSikora) [reply CHANGELOG.md]: Good catch, thanks!

File: CHANGELOG.md (hunk 3 above)
Comment (PiotrSikora) [reply CHANGELOG.md:9]: Good question, but I don't this so. It doesn't exist on its own, and we'd be adding noise into change log.

Comment (mpwarres) [reply CHANGELOG.md:9]: Yeah, makes sense.

**Pull request merged**